tensorflow/python/__init__.py

```diff
@@ -36,9 +36,9 @@
 
 # go/tf-wildcard-import
 # pylint: disable=wildcard-import,g-bad-import-order,g-import-not-at-top
-from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow
 
 from tensorflow.python.eager import context
+from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow
```

Comment:
Just checking, do you need the reorder here? If you do, this is all good.

Reply:
This is due to the following modification, we find that pywrap_dlopen_global_flags.py was built, and if pywrap_tensorflow imported before eager.context, it will get crash in tensor_shape_pb2.py(`_descriptor.FileDescriptor` func) 
   ```
  # modular TF need this file to load expose C API symbols
        # in pywrap_tensorflow_internal.so
        "pywrap_dlopen_global_flags.py",
    ],
```
crashed in
```
DESCRIPTOR = _descriptor.FileDescriptor(
  name='tensorflow/core/framework/tensor_shape.proto',
  package='tensorflow',
  syntax='proto3',
  serialized_options=_b('\n\030org.tensorflow.frameworkB\021TensorShapeProtosP\001ZSgithub.com/tensorflow/tensorflow/tensorflow/go/core/framework/           tensor_shape_go_proto\370\001\001'),
  serialized_pb=_b('\n,tensorflow/core/framework/tensor_shape.proto\x12\ntensorflow\"z\n\x10TensorShapeProto\x12-\n\x03\x64im\x18\x02 \x03(\x0b\x32 .        tensorflow.TensorShapeProto.Dim\x12\x14\n\x0cunknown_rank\x18\x03 \x01(\x08\x1a!\n\x03\x44im\x12\x0c\n\x04size\x18\x01 \x01(\x03\x12\x0c\n\x04name\x18\x02   \x01(\tB\x87\x01\n\x18org.tensorflow.frameworkB\x11TensorShapeProtosP\x01ZSgithub.com/tensorflow/tensorflow/tensorflow/go/core/framework/                    tensor_shape_go_proto\xf8\x01\x01\x62\x06proto3')
)    
```

Reply:
Interesting. So long as its needed, the new order is good to keep.